<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>649</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>938</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QFormLayout" name="formLayout_4">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_17">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_18">
        <item>
         <widget class="QLabel" name="label_18">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <underline>true</underline>
           </font>
          </property>
          <property name="text">
           <string>Область изобретения:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="electronic">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Электроника и ВТ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="chemistry">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Химия</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="phisics">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Физика</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="biology">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Биология</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="maths">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Математика</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_19">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <underline>true</underline>
           </font>
          </property>
          <property name="text">
           <string>Наименование:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="name">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_20">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <underline>true</underline>
           </font>
          </property>
          <property name="text">
           <string>Год:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="year">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="minimum">
           <number>1900</number>
          </property>
          <property name="maximum">
           <number>2022</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_21">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <underline>true</underline>
           </font>
          </property>
          <property name="text">
           <string>Авторы:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="string">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="nobel">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Присуждена нобелевская премия</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_19">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_20"/>
        </item>
        <item>
         <widget class="QCheckBox" name="patent">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Наличие патента:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_21">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="first_data">
              <property name="text">
               <string>Выбрать запись 1</string>
              </property>
              <property name="default">
               <bool>false</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="second_data">
              <property name="text">
               <string>Выбрать запись 2</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_22">
            <item>
             <widget class="QPushButton" name="save">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Сохранить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="show">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Показать</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Патент</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_23">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_24">
           <item>
            <widget class="QLabel" name="label_23">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <underline>true</underline>
              </font>
             </property>
             <property name="text">
              <string>Дата регистрации патента:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="date">
             <property name="maximumDateTime">
              <datetime>
               <hour>23</hour>
               <minute>59</minute>
               <second>59</second>
               <year>2021</year>
               <month>12</month>
               <day>31</day>
              </datetime>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string> Область изобретения:                                              Год:                                              Наименование:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QListWidget" name="list">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="batchSize">
         <number>101</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>208</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="create_1">
        <property name="text">
         <string>Создать</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5"/>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="create_10">
        <property name="text">
         <string>Заполнить</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="redact">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Редактировать</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="delete1">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Искать 3 записи по области:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="search_string">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QPushButton" name="search_in">
        <property name="text">
         <string>Поиск</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <widget class="QLabel" name="info">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>343</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>938</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>mainToolBar</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>date</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>607</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>electronic</sender>
   <signal>clicked(bool)</signal>
   <receiver>nobel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chemistry</sender>
   <signal>clicked(bool)</signal>
   <receiver>nobel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>phisics</sender>
   <signal>clicked(bool)</signal>
   <receiver>nobel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>387</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>biology</sender>
   <signal>clicked(bool)</signal>
   <receiver>nobel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maths</sender>
   <signal>clicked(bool)</signal>
   <receiver>nobel</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>patent</sender>
   <signal>clicked(bool)</signal>
   <receiver>date</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>607</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>patent</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>550</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mainToolBar</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>nobel</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mainToolBar</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>second_data</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mainToolBar</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>first_data</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>23</x>
     <y>452</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
